### PR TITLE
feat: migration guide

### LINKED ISSUE #2594: Check if can set a default value if some value is not present in materialized column
Right now if key is not present the default value is set by clickhouse

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes https://github.com/SigNoz/signoz/issues/2594

### CODE REVIEW

**Comment by ankit01-oss** [comment]
@nityanandagohain let me know if this can be merged.

**Comment by nityanandagohain** [comment]
@ankit01-oss we can merge this only after new release is done. I have the merge access, will merge once done.

**Comment by ankit01-oss** [comment]
sure.

File: docs/operate/migration/upgrade-0.27.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+id: upgrade-0.27
+title: Upgrade to 0.27
+sidebar_label: Upgrade to 0.27
+---
+
+# Upgrade to v0.27 from earlier versions (Kubernetes)
+
+In the SigNoz version `>=0.27` i.e. SigNoz chart version `>=0.19.0`, [clickhouse][1]
```
Comment (prashant-shahi) [inline docs/operate/migration/upgrade-0.27.md]: New chart version will be `0.23.x`.

```suggestion
In the SigNoz version `>=0.27` i.e. SigNoz chart version `>=0.23.0`, [clickhouse][1]
```

File: docs/operate/migration/upgrade-0.27.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+is upgraded from version `22.8.8` to `23.7.3`.
+
+This upgrade brings changes in how we index attributes in logs. From now you can have fields with same names but different dataType as selected(indexed) fields.
+
+## First upgrade to v0.27
+
+Follow the platform specific instructions to upgrade to 0.10 and above.
```
Comment (prashant-shahi) [inline docs/operate/migration/upgrade-0.27.md]:
```suggestion
Follow the platform specific instructions to upgrade to 0.27 and above.
```

**Review by prashant-shahi — CHANGES_REQUESTED** [review]

**Review by prashant-shahi — APPROVED** [review]

**Pull request merged**